mmv1/third_party/terraform/utils/dataproc_metastore_service_diff_supress.go.erb

```diff
@@ -0,0 +1,26 @@
+package google
```

Comment:
I think this needs an autogen exception?

Reply:
Question, what exactly does the autogen exception do ? Adding it, just for my knowledge